README.ja.md

```diff
@@ -3172,33 +3466,52 @@ Rubyコミュニティ内でもスタイルについての統一見解が存在
   end
   ```
 
-## 文字列
+## 数値
 
-* <a name="pad-string-interpolation"></a>
-  文字列連結の代わりに文字列挿入や文字列整形を使いましょう。
-<sup>[[link](#pad-string-interpolation)]</sup>
+* <a name="integer-type-checking"></a>
+  整数型の型をチェックする為に、`Integer`を使用して下さい。
+  `Fixnum`はプラットフォームに依存するため、32ビットマシンと64ビットマシンで異なる結果が返されます。
+<sup>[[link](#integer-type-checking)]</sup>
 
   ```Ruby
-  # 悪い例
-  email_with_name = user.name + ' <' + user.email + '>'
+  timestamp = Time.now.to_i
 
-  # 良い例
-  email_with_name = "#{user.name} <#{user.email}>"
+  # 悪い例
+  timestamp.is_a? Fixnum
+  timestamp.is_a? Bignum
 
   # 良い例
-  email_with_name = format('%s <%s>', user.name, user.email)
+  timestamp.is_a? Integer
   ```
 
+  * <a name="random-numbers"></a>
+  オフセットを持つ乱数を生成する場合は、整数の代わりに範囲リテラルを使用しましょう。
+  意図が明確であるためです。サイコロの役割をシミュレートすることを想像してください。
```

Comment:
「オフセットを持つ乱数」は違うと思います。

```
( Prefer to use 
    ( ranges ※)
  instead of
    ( integers with offsets ) , ...)

※:     when generating random numbers
```

こういう構造の文だと思います。「整数とオフセット」のかわりに「範囲」を使いましょうということ。